flows/actions/send_msg.go

```diff
@@ -122,7 +122,18 @@ func (a *SendMsgAction) Execute(run flows.Run, step flows.Step, logModifier flow
 				}
 
 				evaluatedText = translation.Substitute(evaluatedVariables)
-				templating = flows.NewMsgTemplating(a.Templating.Template, evaluatedVariables, translation.Namespace())
+
+				params := make(map[string][]flows.TemplateParam, 1)
+
+				// TODO add support for params in other components besides body
+				if len(evaluatedVariables) > 0 {
```

Comment:
I think here is the right place since we can have access to the param types in the template translations once we support other components and parameter types